library/src/main/java/com/elvishew/xlog/printer/file/FilePrinter.java

```diff
@@ -142,6 +150,19 @@ void doPrintln(int logLevel, String tag, String msg) {
     writer.appendLog(flattenedLog);
   }
 
+  /**
+   * Check weather file should be clean
+   */
+  void checkFileClean() {
```

Comment:
Need to be `private`, and use a more reasonable name instead, like `cleanLogFilesIfNecessary`, and also change the javadoc of this method